src/main/java/net/sf/jabref/gui/desktop/JabRefDesktop.java

```diff
@@ -316,26 +316,20 @@ public static void openConsole(File file) throws IOException {
             if (!command.isEmpty()) {
                 command = command.replaceAll("\\s+", " "); // normalize white spaces
                 String[] subcommands = command.split(" ");
-                StringBuilder commandLoggingText = new StringBuilder();
-
-                for (int i = 0; i < subcommands.length; i++) {
-                    subcommands[i] = subcommands[i].replace("%DIR", absolutePath); // replace the placeholder if used
-                    commandLoggingText.append(subcommands[i]);
-                    if (i < (subcommands.length - 1)) {
-                        commandLoggingText.append(" ");
-                    }
-                }
 
-                JabRefGUI.getMainFrame().output(Localization.lang("Executing command \"%0\"...", commandLoggingText.toString()));
-                LOGGER.info("Executing command \"" + commandLoggingText.toString() + "\"...");
+                // replace the placeholder if used
+                String commandLoggingText = command.replace("%DIR", absolutePath);
```

Comment:
Should be enough to replace on string level rather than each space-separated sub-string.